README.md

```
@@ -106,3 +106,81 @@ Expo Android
  ┃ ┗ 📂project element
 
 ```
+
+# :people_holding_hands: Collaborate
+
+### Git 전략
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)